common/scala/src/main/resources/application.conf

```diff
@@ -106,6 +106,23 @@ whisk {
         activations-ddoc = "whisks.v2.1.0"
         activations-filter-ddoc = "whisks-filters.v2.1.0"
     }
+
+    # CouchDB related configuration
```

Comment:
How do these settings relate to the design-document settings above? Those are couchdb specific as well, aren't they?

Reply:
The design docs are used by layers above `ArtifactStore` and hence not internal to it. Infact the view names are used for other [custom stores][1] to determine how to frame the queries.

So they need to be kept separately 

[1]: https://github.com/chetanmeh/openwhisk-mongo/blob/master/common/mongo/src/main/scala/whisk/core/database/mongo/MongoViewMapper.scala